examples/web/nextjs-chat/src/app/api/momento/token/route.ts

```diff
@@ -19,14 +19,27 @@ const authClient = new AuthClient({
 });
 
 export const revalidate = 0;
+
 export async function GET(_request: Request) {
+  // eslint-disable-next-line @typescript-eslint/ban-ts-comment
+  // @ts-ignore
+  const usernameValue = _request.cookies._parsed.get("username").value;
```

Comment:
for some reason TS doesn't think `cookies` is a part of `_request` but i tested this and it works

Reply:
I suspect that you can handle this via next-auth's session stuff but @bruuuuuuuce can probably provide a more clear answer.